code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,421 @@
+/datum/tutorial_base/regional/flock
+	name = "Flock tutorial"
+	var/mob/living/intangible/flock/flockmind/fowner = null
+	region_type = /datum/mapPrefab/allocated/flock_tutorial
+	var/obj/landmark/center = null
+
+	New(mob/M)
+		. = ..()
+		src.AddStep(new /datum/tutorialStep/flock/deploy)
+		src.AddStep(new /datum/tutorialStep/flock/gatecrash)
+		src.AddStep(new /datum/tutorialStep/flock/move)
+		src.AddStep(new /datum/tutorialStep/flock/control)
+		src.AddStep(new /datum/tutorialStep/flock/gather)
+		src.AddStep(new /datum/tutorialStep/flock/convert_window)
+		src.AddStep(new /datum/tutorialStep/flock/floorrun)
+		src.AddStep(new /datum/tutorialStep/flock/release_drone)
+		src.AddStep(new /datum/tutorialStep/flock/kill)
+		src.AddStep(new /datum/tutorialStep/flock/place_sentinel)
+		src.AddStep(new /datum/tutorialStep/flock/deposit_sentinel)
+		src.AddStep(new /datum/tutorialStep/flock/build_thing/finish_sentinel)
+		src.AddStep(new /datum/tutorialStep/flock/build_thing/interceptor)
+		src.AddStep(new /datum/tutorialStep/flock/turret_demo)
+		src.AddStep(new /datum/tutorialStep/flock/showcase)
+		src.exit_point = pick_landmark(LANDMARK_OBSERVER)
+		for(var/turf/T in landmarks[LANDMARK_TUTORIAL_FLOCKCONVERSION])
+			if(src.region.turf_in_region(T))
+				center = T
+				break
+		if (!center)
+			CRASH("Okay who removed the goddamn [LANDMARK_TUTORIAL_FLOCKCONVERSION] landmark")
+		src.fowner = M
+
+	Finish()
+		. = ..()
+		if (!.)
+			return FALSE
+		fowner.reset()
+		fowner.flock.perish(FALSE)
```

Comment:
I think this will kill the Flocktrace that spawns for late join Flockminds, and it will still allow the stats accumulated during the tutorial to show in the Flock control panel stats

Reply:
Could maybe just prevent Trace deaths if death isn't final